util/tools.go

```diff
@@ -19,28 +21,51 @@ func CreateSignalChannel() chan os.Signal {
 	return signalChannel
 }
 
-// CopyStringsMap - returns copy of the map passed as parameter
-func CopyStringsMap(m map[string]string) map[string]string {
-	result := make(map[string]string)
+// CopyStringsMapTo - copies source map to dest which has to be rpc.Message using predefined mapping of keys
+// to message fields
+// PLEASE NOTE: if mapping of key is not present, the key will be ignored and warning logged
+func CopyStringsMapTo(source map[string]string, target *rpc.Message) {
+	for key, val := range source {
+		switch key {
+<<<<<<< HEAD
```

Comment:
Probably a left over of resolving merge conflict

Reply:
Yes, I never saved my resolution...